### DISCUSSION TITLE
[其他]: 在Huggingface里复制到自己的空间后没有任何显示

### DISCUSSION THREAD

**Discussion by andykeppel** [GaiZhenbiao/ChuanhuChatGPT#653] [Q&A]
### 是否已存在现有反馈与解答？

- [X] 我确认没有已有issue或discussion，且已阅读**常见问题**。

### 错误描述

点击Duplicate Space之后App显示running但是还是白板一片（如图）。

已经尝试过Restart Space和Factory reboot，不知道是哪里出了问题。

<img width="1440" alt="截屏2023-04-14 14 13 18" src="https://user-images.githubusercontent.com/62137704/231958535-3d523bbc-dc16-479b-b379-258b4dea0ea1.png">

### 复现操作

点击Duplicate Space

### 错误日志

===== Application Startup at 2023-04-14 06:06:22 =====

9xk86 2023-04-14T06:06:33.753Z 2023-04-14 08:06:33,752 [INFO] [utils.py:313] 获取模板文件名列表
9xk86 2023-04-14T06:06:33.753Z 2023-04-14 08:06:33,752 [INFO] [utils.py:264] 获取文件名列表，目录为templates，文件类型为['.csv', 'json']，是否为纯文本列表True
9xk86 2023-04-14T06:06:33.753Z 2023-04-14 08:06:33,753 [INFO] [utils.py:287] 加载模板文件1 中文提示词.json，模式为2（0为返回字典和下拉菜单，1为返回下拉菜单，2为返回字典）
9xk86 2023-04-14T06:06:33.753Z 2023-04-14 08:06:33,753 [INFO] [utils.py:289] Loading template...
9xk86 2023-04-14T06:06:33.875Z 2023-04-14 08:06:33,875 [WARNING] [utils.py:388] 无法获取IP地址信息。
9xk86 2023-04-14T06:06:33.875Z {'error': True, 'reason': 'RateLimited', 'message': 'Visit https://ipapi.co/ratelimited/ for details'}
9xk86 2023-04-14T06:06:33.886Z /home/user/.local/lib/python3.8/site-packages/gradio/deprecation.py:43: UserWarning: You have unused kwarg parameters in Files, please remove them: {'multiple': True}
9xk86 2023-04-14T06:06:33.886Z   warnings.warn(
9xk86 2023-04-14T06:06:33.886Z 2023-04-14 08:06:33,886 [INFO] [utils.py:313] 获取模板文件名列表
9xk86 2023-04-14T06:06:33.886Z 2023-04-14 08:06:33,886 [INFO] [utils.py:264] 获取文件名列表，目录为templates，文件类型为['.csv', 'json']，是否为纯文本列表True
9xk86 2023-04-14T06:06:33.887Z 2023-04-14 08:06:33,887 [INFO] [utils.py:313] 获取模板文件名列表
9xk86 2023-04-14T06:06:33.887Z 2023-04-14 08:06:33,887 [INFO] [utils.py:264] 获取文件名列表，目录为templates，文件类型为['.csv', 'json']，是否为纯文本列表True
9xk86 2023-04-14T06:06:33.887Z 2023-04-14 08:06:33,887 [INFO] [utils.py:313] 获取模板文件名列表
9xk86 2023-04-14T06:06:33.887Z 2023-04-14 08:06:33,887 [INFO] [utils.py:264] 获取文件名列表，目录为templates，文件类型为['.csv', 'json']，是否为纯文本列表True
9xk86 2023-04-14T06:06:33.887Z 2023-04-14 08:06:33,887 [INFO] [utils.py:287] 加载模板文件1 中文提示词.json，模式为1（0为返回字典和下拉菜单，1为返回下拉菜单，2为返回字典）
9xk86 2023-04-14T06:06:33.887Z 2023-04-14 08:06:33,887 [INFO] [utils.py:289] Loading template...
9xk86 2023-04-14T06:06:33.898Z 2023-04-14 08:06:33,898 [INFO] [utils.py:282] 从用户  中获取历史记录文件名列表
9xk86 2023-04-14T06:06:33.898Z 2023-04-14 08:06:33,898 [INFO] [utils.py:264] 获取文件名列表，目录为history，文件类型为['.json']，是否为纯文本列表True
9xk86 2023-04-14T06:06:33.899Z 2023-04-14 08:06:33,899 [INFO] [utils.py:282] 从用户  中获取历史记录文件名列表
9xk86 2023-04-14T06:06:33.899Z 2023-04-14 08:06:33,899 [INFO] [utils.py:264] 获取文件名列表，目录为history，文件类型为['.json']，是否为纯文本列表True
9xk86 2023-04-14T06:06:33.957Z 2023-04-14 08:06:33,957 [INFO] [ChuanhuChatbot.py:382] 
9xk86 2023-04-14T06:06:33.957Z 川虎的温馨提示：访问 http://localhost:7860 查看界面
9xk86 2023-04-14T06:06:33.958Z Reloading javascript...
9xk86 2023-04-14T06:06:34.032Z Running on local URL:  http://0.0.0.0:7860
9xk86 2023-04-14T06:06:34.032Z 
9xk86 2023-04-14T06:06:34.032Z To create a public link, set `share=True` in `launch()`.

### 运行环境

- OS: Huggingface
- Browser: Chrome
- Gradio version: 
- Python version: 
(或您的其他运行环境信息)

**Comment by GaiZhenbiao** [comment]
我觉得是HuggingFace的问题……今天我试图更新一下HG上的川虎Chat的版本，也是不行

**Comment by GaiZhenbiao** [comment]
我又行了。你再试试吧
